keel-sql/src/main/resources/db/changelog/20201112-add-verification-repository.yml

```diff
@@ -0,0 +1,55 @@
+databaseChangeLog:
+- changeSet:
+    id: add-verification-repository
+    author: fletch
+    changes:
+    - createTable:
+        tableName: verification_state
+        columns:
+          - column:
+              name: environment_uid
```

Comment:
Maybe add FKs for these as we discussed?

Reply:
Seperate PR